src/Products/MeetingCharleroi/migrations/migrate_to_4202.py

```
@@ -0,0 +1,86 @@
+            values = getattr(cfg, attr_name)
+            if "suffix_proposing_group_prereviewers" in values:
+                values.remove("suffix_proposing_group_prereviewers")
+            if "suffix_profile_prereviewers" in values:
+                values.remove("suffix_profile_prereviewers")
+            setattr(cfg, attr_name, values)
+        self.reloadMeetingConfigs(full=True, cfg_ids=["meeting-config-college"])
+        logger.info('Done.')
+
+    def run(self,
+            profile_name=u'profile-Products.MeetingCharleroi:default',
+            extra_omitted=[]):
```

Ruff (0.15.21)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)